File: app/src/main/java/com/example/android/test_android/MainActivity.java
package com.example.android.test_android;

import androidx.appcompat.app.AppCompatActivity;
import androidx.viewpager.widget.ViewPager;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {
private int mode;
private static int count = 1;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_adapter);

        ViewPager pager=findViewById(R.id.pager);
        pager.setAdapter(new MyAdapter(getSupportFragmentManager()));
    }

    public void createNewActivity(View view){
        count++;
        Intent intent = new Intent(this, MainActivity.class);
//        Bundle b = new Bundle();
//        b.putInt("LayoutIndex", mode);
//        intent.putExtras(b);
        startActivity(intent);

    }

    public void createNewNotification(View view){

    }

    public static int getCount() {
        return count;
    }
}

File: app/src/main/java/com/example/android/test_android/MainActivity.kt
package com.example.android.test_android

import androidx.appcompat.app.AppCompatActivity

import android.os.Bundle
import android.view.View
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val adapter = ViewPagerFragmentStateAdapter(supportFragmentManager)
        viewPager.adapter = adapter

        fabPlus.setOnClickListener {
            val itemNo = adapter.inc()
            if(itemNo > 1){
                fabMinus.visibility = View.VISIBLE
            }
            viewPager.setCurrentItem(itemNo.minus(1), true)
        }

        fabMinus.setOnClickListener {
            if(adapter.dec() == 1){
                it.visibility = View.GONE
            }
        }
    }

}
